espanol/matematico/ysoymatematico.md

```diff
@@ -0,0 +1,9 @@
+Y de pronto record� que era matem�tico
+segu�a pensando
+no pod�a detener esto ahora, era demasiado tarde:
+
+�Seguir� con esto? 
+
+[S�.](english/marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.